pkg/controller/atlasproject/cloud_provider_integration.go

```diff
@@ -6,7 +6,9 @@ import (
 	"fmt"
 	"sort"
 
-	"go.mongodb.org/atlas/mongodbatlas"
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/timeutil"
+
+	"go.mongodb.org/atlas-sdk/v20231115004/admin"
```

Comment:
package sorting, please reformat to add `github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/timeutil` in the set of imports between lines 13-17.